samples/drivers/pmic_shell/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(pmic_shell)
```

Comment:
We usually don't add a sample just for a shell module as it can easily be enabled where needed.

Reply:
Removed sample in latest push